lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -415,10 +415,45 @@ PullImportPaths pullImport(
     return PullImportPaths(addedDagPaths, pulledUfePaths);
 }
 
+//------------------------------------------------------------------------------
+//
+UsdMayaPrimUpdaterRegistry::RegisterItem getUpdaterItem(
```

Comment:
This is refactored code taken from pullCustomize so that discardEdits and others can use teh same logic to decide which prim updater to use.